README.md

```diff
@@ -27,9 +27,17 @@ A Mattermost plugin for Confluence. Supports Confluence Cloud, Server, and Data
 With the Confluence plugin, you can subscribe to a variety of events in Confluence, and specify which channels the associated notifications will appear in. 
 
 - In a Mattermost channel, setup subscriptions to get updates from Confluence and manage your notification subscriptions directly within Mattermost.
-- Notify a channel whenever something occurs on a Confluence object:
-  - Confluence spaces, including those created, updated, deleted, and restored, and those with added comments.
-  - Confluence pages, including those created, updated, deleted, restored, and those with added, deleted, or updated comments.
+- Notify a channel whenever something occurs on a Confluence object, we support the following events:
+  - space_updated
+  - page_created
+  - page_restored
+  - page_trashed
+  - page_updated
+  - comment_created
+  - comment_updated
+- User can only create the subscriptions for the spaces and pages for which they have permissions to and the same for the notifications too. 
+- Admins can change permissions for both mattermost and confluence side from the plugin configuration.
+    ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/90389917/f7e70d82-157b-4dc8-aa0a-868b115f2cb4)
```

Comment:
Also, capitalize the Confluence name in the highlight as its a name
![image](https://github.com/Brightscout/mattermost-plugin-confluence-fork/assets/72438220/c2bb7535-6080-42f1-8db7-1d25debd9960)